ios/RCTMapboxGL/RCTMapboxGL.m

```diff
@@ -106,6 +111,25 @@ - (void)createMap
     [self layoutSubviews];
 }
 
+-(void)createOfflinePack:(MGLCoordinateBounds)bounds style:(NSURL*)style fromZoomLevel:(double)fromZoomLevel toZoomLevel:(double)toZoomLevel name:(NSString*)name
+{
+    id <MGLOfflineRegion> region = [[MGLTilePyramidOfflineRegion alloc] initWithStyleURL:style bounds:bounds fromZoomLevel:fromZoomLevel toZoomLevel:toZoomLevel];
+    
+    NSDictionary *userInfo = @{ @"name": name };
```

Comment:
This is pretty limiting. Clients should be able to store more than just a name with each pack, for instance a date, icon, or some other application-specific data. I think the name parameter should be replaced with an NSDictionary-typed userInfo parameter.